plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -607,6 +521,8 @@ static void ffmpeg_mux_data(void *data, struct encoder_packet *packet)
 
 	if (stopping(stream)) {
 		if (packet->sys_dts_usec >= stream->stop_ts) {
+			printf("\n* * * ffmpeg_mux_data: packet->sys_dts_usec and stream->stop_ts: %d, %d * * *\n",
```

Comment:
Need to print in ffmpeg_hls_mux_data.

Reply:
oh yep! of course. Didn't finish testing here